public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -369,19 +369,28 @@ export function createSceneVariableFromVariableModel(variable: TypedVariableMode
   } else if (variable.type === 'constant') {
     return new ConstantVariable({
       ...commonProperties,
-      description: variable.description,
       value: variable.query,
       skipUrlSync: variable.skipUrlSync,
       hide: variable.hide,
     });
   } else if (variable.type === 'textbox') {
     return new TextBoxVariable({
       ...commonProperties,
-      description: variable.description,
       value: variable.query,
       skipUrlSync: variable.skipUrlSync,
       hide: variable.hide,
     });
+  } else if (variable.type === 'groupby') {
+    return new GroupByVariable({
+      ...commonProperties,
+      datasource: variable.datasource,
+      value: variable.current?.value || [],
+      text: variable.current?.text || [],
+      skipUrlSync: variable.skipUrlSync,
+      hide: variable.hide,
+      // @ts-expect-error
+      defaultOptions: variable.options,
```

Comment:
there's a type mismatch here between MetricFindValue and VariableOption (number vs string | string[]).

Reply:
I was able to create the Variable, but I was not able to save it. Is this intended? Should we hide the option from the variable type picker when creating a variable to not be able to create one?